packages/babel-preset-css-prop/src/index.js

```diff
@@ -13,17 +13,22 @@ export default (
   api,
   { pragma, sourceMap, autoLabel, labelFormat, instances, ...options } = {}
 ) => {
+  const isAutomaticRuntime = options.runtime === 'automatic'
+  console.log({ isAutomaticRuntime })
+  const jsxOptions = isAutomaticRuntime
+    ? { importSource: '@emotion/core' }
+    : { pragma: pragmaName, pragmaFrag: 'React.Fragment' }
```

Comment:
`importSource` and `pragma`/`pragmaFrag` are mutually exclusive and based on the runtime https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#react-automatic-runtime-1